src/System.IO.Pipes/tests/NativeOverlapped.unix.cs

```diff
@@ -0,0 +1,3 @@
+
+// stub implementation of NativeOverlapped to compile windows only tests in Unix build
+internal struct NativeOverlapped {}
```

Comment:
This is ok, but would it be better if the tests using NativeOverlapped were moved into Windows-specific files only compiled into Windows builds?